### DISCUSSION TITLE
Add Introduction to Project Management In Foundations For The Odin Project.

### DISCUSSION THREAD

**Discussion by Justin-Bento** [TheOdinProject/curriculum#23510] [Ideas]
Hey Team, 

I started doing the Odin Project a while a go but I put it on hold for personal reasons. However when I looked back at it a week ago, I noticed the Project Management advice is very basic and does not include questions like:

- What is Time Management? 
- What Is A Project?
- How Do I Set A Project Deadline?
- How Do You Stick To A Schedule?
- How Do I Find The Right Project Method For Me?
- Prioritize Assignments Within The Projects?
- How Do I Stick On Time and Within The Deadline I gave myself?
- What is the difference between a tool and a method

I think having an "Introduction to Project Management" solely focused getting things done in foundations introduction. As would be better for new people starting out; since it separates the technical things with organization and productivity recommendations.

What does everyone think about this suggestion?

**Comment by codyloyd** [comment]
Hi! Thanks for reaching out to us with this suggestion.  At this time we do not think this is something we want to consider as an official lesson addition to our course.

We already cover a lot of it in [this lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/gearing-up).. and we really want to streamline the amount of extra reading students need to do before they actually dig into the code.

Also, we are already working on some other resources that hit this kind of topic.

If you are still eager to contribute, I would suggest checking out the above linked lesson and proposing a couple of minimal additions or edits to that content.  Specifically, I don't think we really need to define `what is time management` or `what is a project`... but if you have a few concrete tips for people organizing their projects, they might fit well alongside that content.

Thanks again!  We really appreciate contributions like this, so I feel a little bad turning you down.  Don't let it stop you from continuing to find and suggest improvements to our content!

**Reply by Justin-Bento** [reply]
Hey thanks for responding.

Streamlining the amount of reading is important. If anything could I add a few links in motivation and mindset as an optional read? As I found these resources useful when working through the Odin Project.
